build.rs

```diff
@@ -0,0 +1,9 @@
+#[cfg(feature = "static")]
+fn main() {
+    println!("cargo:rustc-link-lib=static=v4l2");
+    println!("cargo:rustc-link-lib=static=v4lconvert");
+    println!("cargo:rustc-link-lib=static=jpeg");
```

Comment:
Why should it be included here?

Reply:
libv4l2 requires libv4lconvert and libjpeg to be linked as well.